docs/compute.md

```diff
@@ -80,8 +80,23 @@ To remove any passthrough devices from dom0:
 /opt/xensource/libexec/xen-cmdline --delete-dom0 xen-pciback.hide
 ```
 
+:::caution NVMe storage devices on Linux
```

Comment:
I think the whole section should be made an optional `### 3.`, and the next one renumbered to `### 4.`, etc.

Reply:
Done